src/backend/DocDownloaderAPI/pom.xml

```diff
@@ -187,6 +187,11 @@
 			<groupId>ch.qos.logback</groupId>
 			<artifactId>logback-core</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Hi Kyle, Curious why the use of this dependency? I have never had to use this one in the past.

Reply:
This is the access logback part - there was a previous lib that was used with older versions of spring I think.  I was having issues at runtime without this.